index.js

```diff
@@ -761,6 +761,34 @@ class GithubScm extends Scm {
             sha: pullRequestInfo.head.sha
         }));
     }
+
+    /**
+     * Get a name of scm context (e.g. github.com)
+     * @mehod getScmContext
+     * @return {Promise}
+     */
+    _getScmContext() {
+        const contextName = this.config.gheHost ? this.config.gheHost : 'github.com';
+
+        return Promise.resolve(contextName);
+    }
+
+    /**
+     * Determin a scm module can handle a specified scm url
+     * @mehod canHandleUrl
+     * @param {Object}    config
+     * @param {String}    scmUrl
+     * @return {Promise}
+     */
+    _canHandleUrl(config) {
+        return new Promise((resolve) => {
+            resolve(getInfo(config.scmUri));
+        }).then((scmInfo) => {
+            const moduleHost = this.config.gheHost ? this.config.gheHost : 'github.com';
+
+            return scmInfo.host === moduleHost;
+        });
```

Comment:
We removed the canHandleUrl.
https://github.com/screwdriver-cd/screwdriver/issues/365#issuecomment-314705743
https://github.com/screwdriver-cd/scm-base/pull/43